### ISSUE TITLE
Accepting a summon while flying.

### ISSUE THREAD

**Issue by Cabro** [elysium-project/server#2803]
We all know that flying is a bit bugged. At some point, after 2 zones flew by you start to not see mobs/gobjects anymore. If you get a summon while you are in that state, after you accept the summon, you will be teleported at the warlock's location, but everything will be phased. You won't change channels, gobjects will not load, nor NPCs or party members, you also cannot enter instances.
Restarting client does nothing, can't unstuck either.

This effectively makes it impossible for me to play.

**Comment by Spionida** [comment]
A GM can rescue you. I tell people not to accept summons while flying.

**Comment by Cabro** [comment]
Yes, but this is a core bug that should be addressed, not bother GMs, they have enough on their hands :)

**Comment by thepparker** [comment]
Relog or get summoned again

**Comment by Spionida** [comment]
Does not work often
